adops/step-by-step.md

```diff
@@ -1,9 +1,241 @@
-title: Setup Line items For Google Ad Manager
-head_title: Step by step guide to Google Ad Manager Setup
-description: Setting up GAM for Prebid.js
+title: Google Ad Manager with Prebid Step by Step
+head_title: Google Ad Manager with Prebid Step by Step
+description: Step-by-step instructions for setting up line items in GAM for Prebid.
 #note the sidebar type needs to reflect the section this file is displayed in. See _data/sidenav.yml for the side nav categories.
 sidebarType: 3
 ---
-{% include /adops/adops-gam-setup.html %}
+
+# Google Ad Manager with Prebid Step by Step
+{: .no_toc }
+
+* TOC
+{: toc }
+
+This guide contains step-by-step instructions for manually setting up line items in Google Ad Manager (GAM) to work with Prebid. These instructions describe only the specific settings required for Prebid, they are not intended to be comprehensive instructions that replace or duplicate the [GAM documentation](https://support.google.com/admanager#topic=7505988).
+
+Because integrating with Prebid could mean having to create thousands of line items, most companies will automate these steps. We’re showing them here so you can manually create the line items if you need or want to, and also to provide context for the automation.
+
+{: .alert.alert-success :}
+Prebid provides a script you can use to automate these steps: [Prebid Line Item Manager](/tools/line-item-manager.html).
+
+## Prerequisites
+
+Before you begin, we recommend you read through our [Planning Guide](/adops/adops-planning-guide.html) to make sure you know what your configuration is going to look like and you’ve thoroughly documented your decisions.
+
+### Create Advertisers and Orders
+
+GAM works as a hierarchical structure, where line items are children of orders, and orders are children of advertisers. You must have your advertisers and orders set up before you can start creating line items and creatives. The advertisers you create for Prebid will typically depend on whether you’re sending all bids or only the top price bid to the ad server.
+
+- Send Top Bid: Create one general Prebid advertiser
+- Send All Bids: Create one Prebid advertiser per bidder where Orders are organized by bidder, with one or more orders containing line items targeted towards a single bidder.
+
+![Google Ad Manager hierarchy](/assets/images/ad-ops/gam-sbs/gam-hierarchy.png)
+
+
+### Create Native Template
+
+If you’re working with native inventory, you must have your native template created and stored before you begin creating your line item. See [GAM Step by Step - Native Creatives](/adops/gam-native.html).
+
+### Create Keys
+
+When you create your line item, you’ll be targeting key-value pairs that are being sent with the ad request to the ad server. Any keys you target need to be defined in GAM before you can use them in your line items.
+
+To define new keys, in GAM go to **Inventory** > **Key-Values** and enter your Prebid-specific keys, e.g. `hb_pb`, `hb_adid`, `hb_size`, `hb_format`, etc.
```

Comment:
Should a reminder be added here that there is a limit to the length of keys in GAM?

Reply:
Jean added a note below